src/routes/categories/components/CategoryListItem.js

```diff
@@ -8,7 +8,7 @@ import {
   CategoryCaption,
   CategoryThumbnail, Row,
   StyledLink, SubCategoryCaption, SubCategoryThumbnail,
-  SubCategoryThumbnailDiv
+  SubCategoryThumbnailDiv, SubCategory
```

Comment:
Not part of this PR, but: What is the difference between `SubCategoryThumbnailDiv` and `SubCategoryThumbnail`?